run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+failed=0
+
+build_all() {
+  sh -c 'rush build \
+    -t truffle-tutorial-hello-world \
+    -t truffle-tutorial-echo \
+    -t truffle-tutorial-token \
+    -t truffle-tutorial-nft'
+}
+
+rebuild_all() {
+  sh -c 'rush rebuild \
+    -t truffle-tutorial-hello-world \
+    -t truffle-tutorial-echo \
+    -t truffle-tutorial-token \
+    -t truffle-tutorial-nft'
+}
+
+test_all() {
+  examples=(
+    "hello-world"
+    "echo"
+    "token"
+    "nft"
+  )
+
+  ROOT=$(pwd)
+
+  for e in "${examples[@]}"
+  do
+    echo "--------------- testing truffle ${e} ---------------"
```

Comment:
Much like in Waffle tutorials, I'm thinking that we could make the tests just a bit more verbose

```suggestion
    echo "--------------- Running Truffle tests ${e} ---------------"
```